src/components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,23 @@
+import { ThemeSize } from "@src/theme";
+import { observer } from "mobx-react-lite";
+import "../avatar/Avatar.scss"
```

Comment:
Zakaj si šel nazaj in pol spet v isti folder kjer si trenutno?

Reply:
Aja, pomota. Po nepotrebnem. Sem spremeno na "./Avatar.scss"